app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -451,6 +451,7 @@ private void checkPermissionsAndShowNearbyCardView() {
             onLocationPermissionGranted();
         } else if (shouldShowRequestPermissionRationale(Manifest.permission.ACCESS_FINE_LOCATION)
                 && store.getBoolean("displayLocationPermissionForCardView", true)
+                && !store.getBoolean("DoNotAskForLocationPermission", false)
```

Comment:
I checked our repository and seems like we don't have an accurate convention for naming stored variables. I think it should be in lower camel case (doNotAskForLocationPermission). The only exception in Google's Java style guide seems like CONSTANT_CASE to use underscore, which is not our case.

Reply:
Understood Nes, thanks so much! :-)